tests/pylint/runpylint.py

```diff
@@ -16,7 +16,8 @@ def __init__(self):
                                FalsePositive(r"Method 'do_task' is abstract in class 'UnimplementedTask' but is not overridden"),
```

Comment:
*create* in the commit message.

Reply:
Fixed.

Reply:
I still see "create" in the commit message.

Reply:
I originally had *create**s*** there and I've changed it to *create*.

Reply:
Sorry, for some reason I misunderstood it when I read it yesterday. It does indeed look good as it is.